join.spec.md

```diff
@@ -177,7 +178,7 @@ type Query {
   fieldB: X @join__field(graph: B)
 }
 
-type X @join__owner(graph: B) {
+type X @join__owner(graph: B) @join__type(graph: B, key: "usuallyBField") {
```

Comment:
Something about providing a key doesn't seem right to me. It might be fine, it's also an interesting possible edge case with possibly undefined behavior (in our implementation) that I haven't thought about before.

I don't have an answer for this right now but I think it would be good for us to write an issue to investigate and test this. In the mean time, can we use an example that doesn't provide a field which is a key?

cc @martijnwalraven

Reply:
I actually already changed this to a different example and I'm providing a different field now too :)